### DISCUSSION TITLE
How can I get more search results with a limit number of 1000?

### DISCUSSION THREAD

**Discussion by liwenke1** [community/community#30636] [Apps, API and Webhooks] [answered]
Hi,
i'm trying to search go language related repositories with a "star" count of 19, and I find that there are a total of 1154 repositories, however one search only returns 1000 results at most, so how do i get information on the remaining 154 repositories?

my query URL is:
`https://api.github.com/search/repositories?q=language:go%20stars:19&page=1&per_page=100`

**Comment by BergerAPI** [comment, marked as answer]
According to the official GitHub v3 Documentations this is not possible. The only way you could get the rest is by splitting the query into smaller queries, that return less than 1000 results.

There is no further information or solutions on how to get around this.

![image](https://user-images.githubusercontent.com/58854363/186431547-5803b6ab-5d54-44de-8609-549d64cff690.png)

**Comment by liwenke1** [comment]
Hi, thank you for your help
